doc/source/docs/user_guide/reproject_fiona.rst

```diff
@@ -0,0 +1,87 @@
+Re-projecting with Rasterio/Fiona
```

Comment:
```suggestion
Re-projecting using GDAL with Rasterio and Fiona
```

With the addition of rasterio, this might be a better title.

Reply:
I agree 👍